packages/manager/apps/hub/src/index.js

```
@@ -1,82 +1,16 @@
-import ovhManagerNavbar from '@ovh-ux/manager-navbar';
-import ovhManagerOrderTracking from '@ovh-ux/ng-ovh-order-tracking';
+attachPreloader();
 
-import get from 'lodash/get';
-import has from 'lodash/has';
-import head from 'lodash/head';
-
-import atInternet from './components/at-internet';
-import errorPage from './components/error-page';
-import preload from './components/manager-preload';
-import catalog from './catalog';
-import dashboard from './dashboard';
-
-import routing from './routing';
-import './index.scss';
-import 'ovh-ui-kit/dist/oui.css';
-
-Environment.setRegion(__WEBPACK_REGION__);
-Environment.setVersion(__VERSION__);
-
-angular
-  .module('managerHubApp', [
-    'pascalprecht.translate',
-    atInternet,
-    catalog,
-    dashboard,
-    errorPage,
-    ngOvhUiRouterLineProgress,
-    'pascalprecht.translate',
-    'oui',
-    ovhManagerCore,
-    ovhManagerHub,
-    ovhManagerNavbar,
-    ovhManagerOrderTracking,
-    'pascalprecht.translate',
-    preload,
-    uiRouter,
-  ])
-  .config(
-    /* @ngInject */ ($locationProvider) => $locationProvider.hashPrefix(''),
-  )
-  .config(routing)
-  .run(
-    /* @ngInject */ ($translate, $transitions) => {
-      $transitions.onBefore({ to: 'app.**' }, () => $translate.refresh());
-    },
-  )
-  .run(($translate) => {
-    moment.locale(head($translate.use().split('_')));
-  })
-  .run(
-    /* @ngInject */ ($state) => {
-      $state.defaultErrorHandler((error) => {
-        if (error.type === RejectType.ERROR) {
-          $state.go(
-            'error',
-            {
-              detail: {
-                message: get(error.detail, 'data.message'),
-                code: has(error.detail, 'headers')
-                  ? error.detail.headers('x-ovh-queryId')
-                  : null,
-              },
-            },
-            { location: false },
-          );
-        }
+bootstrapApplication().then(({ region }) => {
+  import(`./config-${region}`)
+    .catch(() => {})
```

i guess it should not occurs but silencing errors is not a good practice i think it makes debugging harder, maybe :
```
.catch(console.error)
```

---

These configuration files are optional so I don't think this is a good idea.